python/ray/serve/batching.py

```diff
@@ -2,11 +2,43 @@
 from functools import wraps
 from inspect import iscoroutinefunction
 import time
-from typing import Any, Callable, List, Optional, overload, Tuple, TypeVar
+from typing import Any, Callable, Dict, List, Optional, overload, Tuple, TypeVar
+from dataclasses import dataclass
 
+from ray._private.signature import extract_signature, flatten_args, recover_args
 from ray.serve.exceptions import RayServeException
 
 
+@dataclass
+class SingleRequest:
+    self_arg: Optional[Any]
+    flat_args: List[Any]
```

Comment:
```suggestion
    flattened_args: List[Any]
```

use consistent naming across the code please, either choose "flattened_args" or "flat_args"